packages/perseus-core/src/widgets/matrix/matrix-util.ts

```diff
@@ -0,0 +1,8 @@
+import type {PerseusMatrixWidgetOptions} from "@khanacademy/perseus-core";
+
+export default function getMatrixPublicWidgetOptions(
+    options: PerseusMatrixWidgetOptions,
+) {
```

Comment:
I think we need the return type added to this file and to this function :) So here it would be `MatrixPublicWidgetOptions`